pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -45,14 +45,10 @@
     "stackprotector"
     "strictoverflow"
   ] ++ lib.optional (with stdenvNoCC;
-    # Musl-based platforms will keep "pie", other platforms will not.
-    # If you change this, make sure to update section `{#sec-hardening-in-nixpkgs}`
```

Comment:
I can't help but notice that this is not updated in this PR. Will it be updated somewhere else?